source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -33,21 +33,21 @@ Global      | [`$confdir`][confdir]`/hiera.yaml`              | `/etc/puppetlabs
 Environment | [`<ENVIRONMENT>`][environment]`/hiera.yaml` | `/etc/puppetlabs/code/environments/production/hiera.yaml` `C:\ProgramData\PuppetLabs\code\environments\production\hiera.yaml`
 Module      | [`<MODULE>`][module]`/hiera.yaml`                           | `/etc/puppetlabs/code/environments/production/modules/ntp/hiera.yaml` `C:\ProgramData\PuppetLabs\code\environments\production\modules\ntp\hiera.yaml`
 
-> Note: You can change the location for the global layer's hiera.yaml with Puppet’s `hiera_config` setting.
+> Note: You can change the location for the global layer's hiera.yaml with Puppet's `hiera_config` setting.
 
-Related topics: [$confdir][confdir], [<MODULE>][module].
+Related topics: [$confdir][confdir], [`<MODULE>`][module].
```

Comment:
The tag-like `<MODULE>` didn't render because it looks like an HTML tag, which broke the link. Wrapping it in preformatting backticks renders it, fixing the link.